
__FILE__ expands to the full pathname during VPATH builds; this
clutters up the debug log (and also bloats binaries very slightly).
The -pie patch defines AP_FNAME during build which is equivalent
to basename(__FILE__); use that for debug logging where available.

AP_FNAME uses $(notdir ...), a GNU make extension, so this can't
go upstream.

Note this patch depends on the pie patch.

--- httpd-2.0.48/include/http_log.h.debuglog
+++ httpd-2.0.48/include/http_log.h
@@ -280,7 +280,11 @@
  * @see ap_log_rerror
  * @see ap_log_cserror
  */
+#ifdef AP_FNAME
+#define APLOG_MARK     AP_FNAME,__LINE__,APLOG_MODULE_INDEX
+#else
 #define APLOG_MARK     __FILE__,__LINE__,APLOG_MODULE_INDEX
+#endif
 
 /**
  * Set up for logging to stderr.

